great-idea/css/index.css

```diff
@@ -170,30 +172,122 @@ h5 {
 	margin: 0auto;
```

Comment:
a better way to do this is 
margin: 0 auto; 
the space makes it easier to read

Reply:
Awesome I will remember that, I thought about doing the margin in one line but I wasn't sure if it was the same as padding, glad that it is makes it easy, I should have googled it, thanks for the feedback.